rskj-core/src/main/java/co/rsk/rest/RestServerInitializer.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rest;
+
+import co.rsk.rest.dto.RestModuleConfigDTO;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.http.HttpServerCodec;
+
+public class RestServerInitializer extends ChannelInitializer<SocketChannel> {
+
+    private final RestModuleConfigDTO restModuleConfigDTO;
+
+    public RestServerInitializer(RestModuleConfigDTO restModuleConfigDTO) {
+        this.restModuleConfigDTO = restModuleConfigDTO;
+    }
+
+    @Override
+    protected void initChannel(SocketChannel ch) {
+        ChannelPipeline pipeline = ch.pipeline();
+        pipeline.addLast("httpServerCodec", new HttpServerCodec());
+        pipeline.addLast("httpServerHandler", new RestServerHandler(restModuleConfigDTO));
```

Comment:
If the logic is increased maybe it would have sense to have an error handler.

Reply:
Yes, we can add it whenever it became necessary, the idea of having this logic here is precisely that: adding some code clarity and separation of concerns. For now, I believe that the different components are taking care of different kind of errors in a place that "makes sense".